config/crd/bases/core.choreo.dev_buildplanes.yaml

```
@@ -0,0 +1,98 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.4
+  name: buildplanes.core.choreo.dev
+spec:
+  group: core.choreo.dev
```

```suggestion
  group: openchoreo.dev
```

Shall we update the CRD's too?